app/products/calls/screens/host_controls/host_controls.tsx

```diff
@@ -63,20 +80,35 @@ export const HostControls = ({
     const stopScreensharePress = useCallback(async () => {
         hostStopScreenshare(currentCall.serverUrl, currentCall.channelId, session.sessionId);
         await dismissBottomSheet();
-    }, [currentCall.serverUrl, currentCall.id, session.sessionId]);
+    }, [currentCall.serverUrl, currentCall.channelId, session.sessionId]);
+
+    const profilePress = useCallback(async () => {
+        dismissBottomSheet();
+        openUserProfile(session);
+    }, [session]);
+
+    const removePress = useCallback(async () => {
+        removeFromCall(currentCall.serverUrl, displayName, currentCall.channelId, session.sessionId, intl);
+        await dismissBottomSheet();
+    }, [displayName, currentCall.serverUrl, currentCall.channelId, session.sessionId]);
 
     const snapPoints = useMemo(() => {
-        const items = 1 + (session.muted ? 0 : 1) + (sharingScreen ? 1 : 0) + (session.raisedHand ? 1 : 0);
+        const items = 3 + (session.muted ? 0 : 1) + (sharingScreen ? 1 : 0) + (session.raisedHand ? 1 : 0);
```

Comment:
why the 3?

Reply:
Make host, View profile, Remove from call.